src/components/Card/Card.js

```diff
@@ -1,12 +1,18 @@
-import React from 'react'
-import './card.css'
+import React from "react";
+import "./card.css";
+import PlaceHolderImage from "../../assets/placeholder_image.png";
+
+const Card = ({ id, fullName, title, photo }) => {
+  const photoURL = `https://codesydney-website.s3-ap-southeast-2.amazonaws.com/mentor/${id}.png`;
 
-const Card = () => {
   return (
-    <div>
-      <h1>CARD COMPONENT</h1>
+    <div className="card">
+      <img className="card-image" src={photo ? photoURL : PlaceHolderImage} alt="user-profile" />
+      <div className="card-fullName">{fullName}</div>
+      <div className="card-title">{title}</div>
+      <button className="card-button">More Info</button>
```

Comment:
We should replace this with description. We can use the in built javascript function if the description is too long. ```{description.slice(0,50)...}``` for whatever the number of characters that look good.